PWGHF/D2H/Tasks/taskSigmacToCascade.cxx

```diff
@@ -10,19 +10,18 @@
 // or submit itself to any jurisdiction.
 
 /// \file taskSigmacToCascade.cxx
-///
-/// \note Task for Σc0,++ → Λc(→ K0sP) + π-,+ analysis
+/// \brief Task for Σc0,++ → Λc(→ K0sP) + π-,+ analysis
 /// \note Here the Lc from the cascade channel is obtained using the task taskLcToK0sP.cxx
 /// \author Rutuparna Rath <rrath@cern.ch>, INFN BOLOGNA and GSI Darmstadt
 /// In collaboration with Andrea Alici <aalici@cern.ch>, INFN BOLOGNA
 
-#include "CommonConstants/PhysicsConstants.h"
 #include "Framework/AnalysisTask.h"
 #include "Framework/HistogramRegistry.h"
 #include "Framework/runDataProcessing.h"
+
+#include "CommonConstants/PhysicsConstants.h"
 #include "Common/Core/TrackSelection.h"
 #include "Common/Core/TrackSelectionDefaults.h"
-
 #include "PWGHF/Core/HfHelper.h"
```

Comment:
Keep the blank line to separate the PWGHF block.

Reply:
Done.